endpointmanager/pkg/chplquerier/vendorquerier_integration_test.go

```diff
@@ -208,13 +215,37 @@ func Test_GetCHPLVendors(t *testing.T) {
 	_, err = store.DB.Exec("DELETE FROM vendors;") // reset values
 	th.Assert(t, err == nil, err)
 
-	err = GetCHPLVendors(ctx, store, &(tc.Client))
-	switch reqErr := errors.Cause(err).(type) {
-	case *url.Error:
-		th.Assert(t, reqErr.Err == context.Canceled, "Expected error stating that context was canceled")
-	default:
-		t.Fatal("Expected context canceled error")
+	err = GetCHPLVendors(ctx, store, &(tc.Client), "")
+	found := false
+	for i := range hook.Entries {
+		if strings.Contains(hook.Entries[i].Message, expectedErr) {
+			found = true
+			break
+		}
+	}
+	th.Assert(t, found, "expected an error to be logged")
+
+	// test http status != 200
+
+	tc = th.NewTestClientWith404()
+	defer tc.Close()
+
+	hook = logtest.NewGlobal()
+	expectedErr = "CHPL request responded with status: 404 Not Found"
```

Comment:
Maybe just to keep things consistent, move the lines with variables hook and expectedErr to above the lines where the test client is made, since this is the ordering in the other files containing these kinds of tests